.github/ISSUE_TEMPLATE/SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security
+
+## Reporting a Bug in F@H Software
+
+To report security vulnerabilities, please email joseph(at)cauldrondevelopment(dot))com
```

Comment:
`(dot))com` should be `(dot)com` I guess? 🙂

Reply:
Thanks for that... clearly, I need more than four eyes :sweat_smile: